misc/allow-block-list/src/lib.rs

```diff
@@ -474,7 +474,7 @@ mod tests {
     #[behaviour(prelude = "libp2p_swarm::derive_prelude")]
     struct Behaviour<S> {
         list: super::Behaviour<S>,
-        keep_alive: libp2p_swarm::keep_alive::Behaviour,
+        dummy_behaviour: libp2p_swarm::dummy::Behaviour,
     }
```

Comment:
This can also be simplified by removing this struct altogether and directly referencing `super::Behaviour<S>` in the tests. You might need to configure an idle connection timeout for the tests to keep passing!